src/services/jsTyping.ts

```diff
@@ -80,31 +83,29 @@ namespace ts.JsTyping {
         }
 
         const filesToWatch: string[] = [];
-        // Directories to search for package.json, bower.json and other typing information
-        let searchDirs: string[] = [];
-        let exclude: string[] = [];
 
-        mergeTypings(typeAcquisition.include);
-        exclude = typeAcquisition.exclude || [];
+        forEach(typeAcquisition.include, addInferredTyping);
+        const exclude = typeAcquisition.exclude || [];
 
-        const possibleSearchDirs = map(fileNames, getDirectoryPath);
-        if (projectRootPath) {
-            possibleSearchDirs.push(projectRootPath);
+        // Directories to search for package.json, bower.json and other typing information
+        const possibleSearchDirs = createMap<true>();
+        for (const f of fileNames) {
+            possibleSearchDirs.set(getDirectoryPath(f), true);
         }
-        searchDirs = deduplicate(possibleSearchDirs);
-        for (const searchDir of searchDirs) {
+        possibleSearchDirs.set(projectRootPath, true);

```

Comment:
At one point I believe there were scenarios where this could be undefined in VS (ex: websites, open folder). Not sure if this is still the case though.